app.js

```diff
@@ -1,15 +1,27 @@
 const express = require("express");
 const app = express();
 const { getTopics } = require("../LB-NC-NEWS/controller/topics.controller");
-const {getEndpoints} = require('../LB-NC-NEWS/controller/endpoints.controller')
+const {
+  getEndpoints,
+} = require("../LB-NC-NEWS/controller/endpoints.controller");
+const { getArticle } = require("../LB-NC-NEWS/controller/articles.controller");
```

Comment:
Exports and require can always use a relative path which might make things a bit nicer. Something like '/controller/articles.controller' would have worked here.